### ISSUE TITLE
Focus trapped on notifications menu

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1430]
### Description

Focus gets lost after I delete a notification

### Steps to reproduce

1. Use the keyboard and go through the notifications menu and delete a notification
2. The focus gets lost ( the arrow keys can't be used and tabbing doesn't work properly)

### Expected behavior

After a notification gets deleted, the focus should move to the next available notification

### Which operating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

-

### Version

5.7.0
